### PR TITLE
Add tertiary phone number

### PR DESCRIPTION
**Pull request by jensschuppe**
... just as #52 did for a secondary phone number, i. e. a `phone3` parameter and configuration for the phone type.

### CODE REVIEW

**Comment by jensschuppe** [comment]
@jaapjansma fancy reviewing this, as you authored the secondary phone thing?

This could also use a test which I haven't had the time adding.

File: CRM/Xcm/Configuration.php [hunk 1]
```diff
@@ -369,6 +369,14 @@ public function secondaryPhoneType() {
     return (int) CRM_Utils_Array::value('secondary_phone_type', $options);
   }
 
+  /**
+   * Get ternary phone type
+   */
+  public function ternaryPhoneType() {
```
Comment (bjendres) [inline CRM/Xcm/Configuration.php:375]: Shouldn't it be "tertiary"?

**Review by bjendres — COMMENTED** [review]
Looks good to me, but I haven't _tested_.

I'm fairly sure it needs to be "tertiary" instead of "ternary", just so we don't confuse English native speakers. But if you don't want to change it, i'll add a "sic" :)

Let me know if you need/want me to actually test it.

File: CRM/Xcm/Configuration.php (hunk 1 above)
Comment (jensschuppe) [reply CRM/Xcm/Configuration.php]: You're right, renamed all occurences.

**Comment by jensschuppe** [comment]
> Let me know if you need/want me to actually test it.

There's a unit test for secondary phone which might be easily adjusted oder copied for testing `phone3`, feel free to add one if you like.

**Comment by bjendres** [comment]
> > Let me know if you need/want me to actually test it.
> 
> @bjendres There's a unit test for secondary phone which might be easily adjusted oder copied for testing `phone3`, feel free to add one if you like.

I did, but I cannot assess whether this copy tests all the right things. But the tests all pass now.

**Pull request merged**